doc/classes/Node.xml

```diff
@@ -679,7 +679,7 @@
 			<param index="0" name="new_parent" type="Node" />
 			<param index="1" name="keep_global_transform" type="bool" default="true" />
 			<description>
-				Changes the parent of this [Node] to the [param new_parent]. The node needs to already have a parent.
+				Changes the parent of this [Node] to the [param new_parent]. The node needs to already have a parent. The node's [code]owner[/code] is preserved, if the owner is or is an ancestor of the new parent.
```

Comment:
"if the owner is or"
?

Reply:
boy i don't know what happened there lol. i think i was just tired. i was trying to go for "if the owner is - or is an ancestor of - the new parent..." but that is extremely poorly worded